transactions/slash-rewards/2024/oct-8/README.md

```diff
@@ -1,6 +1,6 @@
-# Slashing node operator rewards for Node IDs 2b396b7fab0102f104a2af7e095b145cc14da28f863564802e158afc3e07e638 and 1cf032dd5b3cf9a63943e1f04472bb57cc1240d5cdd86c5fde05b4dabf8b1e7a
+# Slashing node operator rewards for Node IDs 2b396b7fab0102f104a2af7e095b145cc14da28f863564802e158afc3e07e638, 1cf032dd5b3cf9a63943e1f04472bb57cc1240d5cdd86c5fde05b4dabf8b1e7a and 6c6af0933b710655ec553f4bead3b01c5e0a3ffd1194ee536efb926b356c54aa.
```

Comment:
Samsungs EN - Node ID: `6c6af0933b710655ec553f4bead3b01c5e0a3ffd1194ee536efb926b356c54aa` is also down.